### PR TITLE
[stable/fluent-bit] add extraConfigMapEntries value

### PR DESCRIPTION
**Pull request by spiette**
#### What this PR does / why we need it:

This option let's you add arbitrary files in /fluent-bit/etc and use them in the configuration. The primary use case is for lua scripts: otherwise you need to create a configMap outside the chart. It is also convenient to use with `@INCLUDE` directives.  

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by alexmontecucco** [comment]
Hello,
Are we proceeding with this change ?
Would be useful to us as well. Only issue I see is that the Chart.yaml file has been updated since this MR was created and is now conflicting.

**Comment by spiette** [comment]
Charts.yaml have been updated

File: stable/fluent-bit/templates/config.yaml [hunk 1]
```diff
@@ -267,5 +267,7 @@ data:
 {{- end }}
 {{ end }}
 {{- end }}
-
+{{- if .Values.extraConfigMapEntries }}
+{{ .Values.extraConfigMapEntries | indent 2 }}
```
Comment (Towmeykaw) [inline stable/fluent-bit/templates/config.yaml:271]: Will this work when adding an object? Should't it be something like {{ toYaml .Values.extraConfigMapEntries | indent 2 }} ?

**Comment by DandyDeveloper** [comment]
@spiette 

How does this differ from just doing the below:

```
  existingConfigMap: elasticsearch-fluentbit-config-custom
  rawConfig: |-
    @INCLUDE fluent-bit-service.conf
    @INCLUDE fluent-bit-input.conf
    @INCLUDE fluent-bit-filter.conf
    @INCLUDE fluent-bit-output.conf
```

Wherein you create your own ConfigMap with the content of what you need under the local `/templates`?

File: stable/fluent-bit/templates/config.yaml (hunk 1 above)
Comment (spiette) [reply stable/fluent-bit/templates/config.yaml:271]: It's a map, so you would use it like this:
```extraConfigMapEntries: |
        scripts.lua: |
          function cb_print(tag, timestamp, record)
            return code, timestamp, record
          end
```

**Comment by spiette** [comment]
@DandyDeveloper 
> Wherein you create your own ConfigMap with the content of what you need under the local `/templates`?
> 
> Adding more options may end up confusing people in the long run IMO.
 
But then you have to manage the configmaps outside the chart, which is very inconvenient.

**Pull request closed without merging**